### DISCUSSION TITLE
Continuation of https://github.com/microsoft/WSL/issues/5125

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WSL#6461] [General]
Given Windows Server 2019 is still running on 17763, the WSLv1 workaround for Ubuntu 20+ is still required. It's not clear why that issue was closed, as no official solution was provided. cc: @therealkenc

https://github.com/microsoft/WSL/issues/5125

**Comment by therealkenc** [comment]
The consensus [seems](https://github.com/microsoft/WSL/issues/5125#issuecomment-633279507) #5125 was dupe #4898, although in 41 posts (at the time) there wasn't a single `strace(1)` gist demonstrating a `CLOCK_REALTIME EINVAL` I could find. In Cyle's gist the `clock_gettime()` succeeds. Academic I guess; does not repro in WSL1 19631 on 20.04. #4898 was addressed in 19603 > 17763 (+ WSL2 natch). There is no official work-around (if there is such a thing), although any solution that avoids firing a `CLOCK_REALTIME` would be plausible.
